src/tests/daos_perf.c

```diff
@@ -1398,10 +1403,25 @@ main(int argc, char **argv)
 	}
 
 	if (ts_mode == TS_MODE_VOS) {
+		if (ts_ctx.tsc_mpi_size > 1 &&
+		    (access("/etc/daos_nvme.conf", F_OK) != -1)) {
+			fprintf(stderr,
+				"no support: multi-proc vos_perf with NVMe\n");
+			return -1;
+		}
 		ts_ctx.tsc_cred_nr = -1; /* VOS can only support sync mode */
-		if (strlen(ts_pmem_file) == 0)
-			strcpy(ts_pmem_file, "/mnt/daos/vos_perf.pmem");
-
+		if (strlen(ts_pmem_file) == 0) {
+			int len = snprintf(NULL, 0, "/mnt/daos/vos_perf%d.pmem",
+					   ts_ctx.tsc_mpi_rank) + 1;
+			snprintf(ts_pmem_file, len, "/mnt/daos/vos_perf%d.pmem",
+				 ts_ctx.tsc_mpi_rank);
+		} else {
+			char id[4];
```

Comment:
The strlen of max int32_t is 5 and NIL. So, It should be 6 size if int type is int32_t. I'd better use 16 or 32 for better alignment here,

Reply:
I just propose the following:
```suggestion
			char id[16];

```

Reply:
Done.